app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/network/MarvelApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.network
```

Comment:
don't like packaging structure, it can be more modern, like domain.layer

http://www.kubrynski.com/2015/11/smart-package-structure-to-improve.html

you are using MVP so layers should be in packages `view` `presenter` `model`